scripts/pre_commit_linter.py

```diff
@@ -821,8 +821,14 @@ def _check_docstrings(all_files):
     summary_messages = []
     files_to_check = [
         filename for filename in all_files if not
-        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)
+        (any(
```

Comment:
I don't think you need the outer parens. Didn't this and the next line all fit in one line before?

Reply:
Done.